system-variables.md

```diff
@@ -933,3 +933,26 @@ explain select * from t where age=5;
 - Scope: SESSION | GLOBAL
 - Default value: OFF
 - This variable controls whether to track the memory usage of the aggregate function. When you enable this feature, TiDB counts the memory usage of the aggregate function, which might cause the overall SQL memory statistics to exceed the threshold [`mem-quota-query`](/tidb-configuration-file.md#mem-quota-query), and then be affected by the behavior defined by [`oom-action`](/tidb-configuration-file.md#oom-action).
+
+### `tidb_enable_async_commit` <span class="version-mark">New in v5.0.0-rc</span>
+
+> **Warning:**
+>
+> `tidb_enable_async_commit` is currently an experimental feature. It is not recommended to use this feature in the production environment. Currently, the following incompatible issues are found, and be aware of them if you need to use this feature:
+
+> - This feature is incompatible with [TiCDC](/ticdc/ticdc-overview.md) and might cause TiCDC to run abnormally.
+> - This feature is incompatible with [Compaction Filter](/tikv-configuration-file.md#enable-compaction-filter). when you use two features at the same time, write loss might occur.
+
+- Scope: SESSION | GLOBAL
+- Default value: OFF
+- This variable controls whether to enable the async commit feature for the second phase of the two-phase transaction commit to perform asynchronously in the background. Enabling this feature can reduce the latency of transaction commit. This feature does not take effect when TiDB Binlog is enabled.
+
+> **Note:**
+>
+> Enabling this feature can not guarantee external consistency of the transactions. For details, refer to [`tidb_guarantee_external_consistency`](#tidb_guarantee_external_consistency-new-in-v500-rc).
```

Comment:
This is also a compatibility warning.

Reply:
"Enabling this feature can not guarantee external consistency" reads strangely to me...

Reply:
Modified